libraries/GCS_MAVLink/GCS_Signing.cpp

```diff
@@ -219,6 +220,7 @@ void GCS_MAVLINK::save_signing_timestamp(bool force_save_now)
         if (status && status->signing && status->signing->timestamp > key.timestamp) {
             key.timestamp = status->signing->timestamp;
             need_save = true;
+            break;
```

Comment:
This one is "wrong". There can be a bigger timestamp on another channel and we want to save the biggest one.

Reply:
Gah.  Seeing patterns where there are none.  Thanks.